documentation/modules/security/con-kafka-listener-certificates.adoc

```diff
@@ -6,10 +6,15 @@
 
 = Kafka listener certificates
 
-Sometimes you might want to leverage your existing security infrastructure, such as your company's private CA or a public CA, when integrating your client applications with {ProductName}. 
-To support this requirement, {ProductName} allows you to provide your own _Kafka listener certificates_ and public keys for external listeners which have TLS encryption enabled.
-Your Kafka client applications will connect to Kafka brokers using Kafka listener certificates rather than CA certificates signed by the cluster CA or clients CA.
+You can provide your own server certificates and CA public keys for the following types of listeners:
```

Comment:
Sorry, I had to get back to this. I do not think the CA belongs here.

```suggestion
You can provide your own server certificates and public keys for the following types of listeners:
```

Reply:
OK, that's updated now.